src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,79 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_KEY = "supply";
+    private static final String BUY_KEY = "buy";
+    private static final int KEY_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+    private int supplyValue = 0;
+    private int buyValue = 0;
```

Comment:
```suggestion
    private int supplyValue;
    private int buyValue;
```

Reply:
You don`t need to init primitive fields by default value

Reply:
++